### DISCUSSION TITLE
should imported less files automatically refresh after saved

### DISCUSSION THREAD

**Discussion by zanona** [parcel-bundler/parcel#5299] [Q&A] [answered]
I have a `index.html` file with the following:

```html
<link ref='stylesheet' href='./src/styles/index.less'/>
```

and under my `src/styles/index.less` I then have:

```less
@import './sub';
```

By making changes and saving `src/styles/sub.less`, should I expect Parcel to hot reload styles?
Apparently I need to save `src/styles/sub.less` and then save `src/styles/index.less` to see changes applied.

**Comment by gmougeolle** [comment, marked as answer]
Experiencing the same issue. I've submitted a PR (#5686) addressing this.
